### PR TITLE
KAFKA-14589 [4/4] Tests of ConsoleGroupCommand rewritten in java

### PR DESCRIPTION
**Pull request by nizhikov**
This PR is part of https://github.com/apache/kafka/pull/14471
Is contains `ResetConsumerGroupOffsetTest` rewritten in java.
Intention of separate PR is to reduce changes and simplify review.

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by chia7712** [comment]
@nizhikov I'm ready to review this PR, so please rebase it. thanks!

**Comment by nizhikov** [comment]
@chia7712 PR merged with the latest trunk and ready for review.

**Comment by nizhikov** [comment]
Checked CI results. Failures unrelated.

File: tools/src/test/java/org/apache/kafka/tools/consumer/group/ResetConsumerGroupOffsetTest.java [hunk 1]
```diff
@@ -0,0 +1,622 @@
...
+ * - One for each scenario, with scope=all-topics
+ * - scope=one topic, scenario=to-earliest
+ * - scope=one topic+partitions, scenario=to-earliest
+ * - scope=topics, scenario=to-earliest
+ * - scope=topics+partitions, scenario=to-earliest
+ * - export/import
+ */
+public class ResetConsumerGroupOffsetTest extends ConsumerGroupCommandTest {
+    @Override
+    public Seq<KafkaConfig> generateConfigs() {
```
Comment (chia7712) [inline tools/src/test/java/org/apache/kafka/tools/consumer/group/ResetConsumerGroupOffsetTest.java]: it seems the config is same as parent? the check of `isNewGroupCoordinatorEnabled` is redundant to `ConsumerGroupCommandTest`.

Comment (nizhikov) [reply tools/src/test/java/org/apache/kafka/tools/consumer/group/ResetConsumerGroupOffsetTest.java]: Fixed.

**Review by chia7712 — APPROVED** [review]
LGTM

all related tests pass on my local.
```sh
./gradlew cleanTest tools:test --tests ResetConsumerGroupOffsetTest --tests ConsumerGroupCommandTest --tests DeleteConsumerGroupsTest --tests DeleteOffsetsConsumerGroupCommandIntegrationTest --tests DescribeConsumerGroupTest
```

**Comment by nizhikov** [comment]
@chia7712 I've checked CI results - looks like failures unrelated to PR changes.

**Comment by chia7712** [comment]
@nizhikov nice patch. Let us go back to #14471 :)

**Pull request merged**